cli/src/chain_spec/mod.rs

```diff
@@ -189,25 +190,22 @@ pub fn config_genesis(network_keys: NetworkKeys, enable_println: bool) -> Genesi
 			},
 			gas_price: 1 * MILLICENTS,
 		}),
-		pallet_sudo: Some(SudoConfig { key: root_key }),
+		pallet_sudo: Some(SudoConfig { key: root_key.clone() }),
 		pallet_babe: Some(BabeConfig { authorities: vec![] }),
 		pallet_im_online: Some(ImOnlineConfig { keys: vec![] }),
 		pallet_authority_discovery: Some(AuthorityDiscoveryConfig { keys: vec![] }),
 		pallet_grandpa: Some(GrandpaConfig { authorities: vec![] }),
 		pallet_membership_Instance1: Some(Default::default()),
 		pallet_treasury: Some(Default::default()),
 		pallet_generic_asset: Some(GenericAssetConfig {
-			assets: vec![
-				CENNZ_ASSET_ID,
-				CENTRAPAY_ASSET_ID,
-				PLUG_ASSET_ID,
-				SYLO_ASSET_ID,
-				CERTI_ASSET_ID,
-				ARDA_ASSET_ID,
-				NEXT_ASSET_ID,
+			assets: vec![CENNZ_ASSET_ID, CENTRAPAY_ASSET_ID],
+			// Grant root key full permissions (mint,burn,update) on the following assets
+			permissions: vec![
+				(CENNZ_ASSET_ID, root_key.clone()),
+				(CENTRAPAY_ASSET_ID, root_key.clone()),
 			],
-			initial_balance: 10u128.pow(18 + 9), // 1 billion token with 18 decimals
-			endowed_accounts: endowed_accounts.clone(),
+			initial_balance: INITIAL_BOND * 2,
```

Comment:
Can set this much higher, previous value will do:  `10u128.pow(18 + 9), // 1 billion token with 18 decimals`

Reply:
done